pkg/services/alerting/notifiers/pagerduty.go

```diff
@@ -79,13 +80,8 @@ type PagerdutyNotifier struct {
 	log         log.Logger
 }
 
-// Notify sends an alert notification to PagerDuty
-func (pn *PagerdutyNotifier) Notify(evalContext *alerting.EvalContext) error {
-
-	if evalContext.Rule.State == models.AlertStateOK && !pn.AutoResolve {
-		pn.log.Info("Not sending a trigger to Pagerduty", "state", evalContext.Rule.State, "auto resolve", pn.AutoResolve)
-		return nil
-	}
+// BuildEventPayload is responsible for building the event payload body for sending to Pagerduty v2 API
+func (pn *PagerdutyNotifier) BuildEventPayload(evalContext *alerting.EvalContext) ([]byte, error) {
```

Comment:
`BuildEventPayload` is only used inside this package so there is no need to be exported.